src/workloads/execution/ProjectSort.yml

```diff
@@ -0,0 +1,121 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: >
+  Check performance of a project followed by a sort that can be swapped in order to push down. Check the 
+  performance both without and without an index.
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 50000
+    BatchSize: &batchSize 100000
```

Comment:
Is the DocumentCount / batchSize mismatch deliberate? I assume it is given that it is reused later in the aggregations.

Reply:
Fixed. I had the lower doc count for faster testing locally